docker/run_web.sh

```diff
@@ -35,21 +35,31 @@ GUNICORN_LOGLEVEL=${GUNICORN_LOGLEVEL:-"info"}
 
 # END GUNICORN CONFIGURATION
 
-echo "Running webserver on http://localhost:${GUNICORN_PORT} ..."
-
-set -x 
-gunicorn \
-    --pythonpath /app \
-    --workers="${GUNICORN_WORKERS}" \
-    --worker-class="${GUNICORN_WORKER_CLASS}" \
-    --worker-connections="${GUNICORN_WORKER_CONNECTIONS}" \
-    --max-requests="${GUNICORN_MAX_REQUESTS}" \
-    --max-requests-jitter="${GUNICORN_MAX_REQUESTS_JITTER}" \
-    --error-logfile=- \
-    --access-logfile=- \
-    --log-file=- \
-    --log-level="${GUNICORN_LOGLEVEL}" \
-    --timeout="${GUNICORN_TIMEOUT}" \
-    --config=python:ichnaea.webapp.gunicorn_settings \
-    --bind 0.0.0.0:"${GUNICORN_PORT}" \
-    ichnaea.webapp.app:wsgi_app
+LOCAL_DEV_ENV=${LOCAL_DEV_ENV:-"False"}
+
+
+if [ "${LOCAL_DEV_ENV}" == "True" ]; then
```

Comment:
We do this kind of thing with Socorro where if we're in local dev environment mode, we run `./manage.py rusnerver` and in server mode run with gunicorn. It makes a bunch of development a lot easier.